1_clusters/modules/hub-registration/variables.tf

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "cluster_name" {
+  description = "The unique name to identify the cluster in ASM."
+  type        = string
+}
+
+variable "cluster_endpoint" {
+  description = "The GKE cluster endpoint."
+  type        = string
+}
+
+variable "project_id" {
+  description = "The project in which the resource belongs."
+  type        = string
+}
+
+variable "location" {
+  description = "The location (zone or region) this cluster has been created in."
+  type        = string
+}
+
+variable "skip_gcloud_download" {
```

Comment:
We're trying to not use this variable any more. https://github.com/terraform-google-modules/terraform-google-gcloud/blob/master/CHANGELOG.md#-breaking-changes